.github/ISSUE_TEMPLATE/dep-monthly.md

```diff
@@ -3,11 +3,6 @@ name: Monthly Run DEP CATS permits
 about: Approve a push of DEP CATS data to production.
```

Comment:
Is there automation to create this issue?

Reply:
Or rather - is there automation for them within this issue? Or is there supposed to be? Like marking publish in edm-data-operations

Reply:
an issue should've been created on a successful run, looks like it wasn't so I'll look into that

based on the github action, looks like the QA used to be assigned a new issue in the old CEQR repo. we should probably make sure the edm-data-operations repo creates an issue for them for this dataset. I'll use the data update issue to note ideas